nana/modules/admins.py

```diff
@@ -480,33 +434,27 @@ async def demote_usr(client, message):
                         can_invite_users=False,
                         can_pin_messages=False,
                     )
-
-
-                    text = "**Demoted**\n"
-                    text += f"User: [{get_mem.user.first_name}](tg://user?id={get_mem.user.id})\n"
-                    text += f"(`{get_mem.user.id}`)\n"
-                    text += f"Chat: `{get_group.title}` (`{chat_id}`)"
-                    await msg(message, text=text)
+                    await message.delete()
                 except ChatAdminRequired:
-                    await msg(message, text="`permission denied`")
+                    await edrep(message, text=tld('denied_permission'))
                     await asyncio.sleep(5)
                     await message.delete()
                     return
 
                 except Exception as e:
-                    await msg(message, text=f"**Log:** `{e}`")
+                    await edrep(message, text=f"**Log:** `{e}`")
                     return
 
             if not message.reply_to_message:
-                await msg(message, text="`reply to a user to demote.`")
+                await edrep(message, text=tld('demote_user'))
                 return
         else:
-            await msg(message, text="``permission denied`")
+            await edrep(message, text="``permission denied`")
     else:
         await message.delete()
 
 
-@app.on_message(Filters.user(AdminSettings) & Filters.command("lock", Command))
+@app.on_message(filters.user(AdminSettings) & filters.command("lock", Command))
```

Comment:
Codacy found an issue: [lock_permission is too complex (20) (MC0001)](https://app.codacy.com/gh/MadeByThePinsHub/Nana-Remix/pullRequest?prid=6330175)